### PR TITLE
Use builder babel config properly

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -155,9 +155,7 @@ module.exports = env => {
           unidiff can be removed once https://github.com/mvoss9000/unidiff/pull/1 is merged */
           exclude: /node_modules(?!\/(@novnc|unidiff))/,
           loader: 'babel-loader',
-          options: {
-            presets: [require.resolve('@theforeman/builder/babel')],
-          },
+          options: require.resolve('@theforeman/builder/babel'),
```
Comment (Ron-Lavi) [inline config/webpack.config.js:158]: [@theforeman/builder/babel](https://github.com/theforeman/foreman-js/blob/master/packages/builder/babel/index.js#L8) returns `({ presets, plugins })`
which indeed should satisfy the `options` here,

although it worked for two years, so I wonder what changed?

**Review by tbrisker — APPROVED** [review]
I assume this is an attempt to fix nightly builds, lets give it a try.

**Comment by tbrisker** [comment]
Gah, I should have looked at the failing test, it doesn't work, reverting

**Comment by Ron-Lavi** [comment]
> Gah, I should have looked at the failing test, it doesn't work, reverting

lol, what was it then? or some guesses

**Comment by ezr-ondrej** [comment]
> Gah, I should have looked at the failing test, it doesn't work, reverting

Sorry :( It was quite desparete try though.

**Pull request merged**